fastlane_bot/config/network.py

```diff
@@ -791,21 +791,21 @@ class _ConfigNetworkSei(ConfigNetwork):
     """
 
     NETWORK = S.NETWORK_SEI
-    NETWORK_ID = "1" # TODO
+    NETWORK_ID = "1329" # TODO
     NETWORK_NAME = "sei"
     DEFAULT_PROVIDER = S.PROVIDER_ALCHEMY
-    RPC_ENDPOINT = "https://evm-rpc.arctic-1.seinetwork.io/" # TODO currently Sei devnet
+    RPC_ENDPOINT = "https://evm-rpc.sei-apis.com/?x-apikey="
```

Comment:
Probably need to update README.md and env.example as well

Reply:
@Lesigh-3100 can you please go over the readme and see that it's clear how to use Sei?

Reply:
Added the `WEB3_SEI` env example to README. It had already been added to .env.example.